### DISCUSSION TITLE
Can i  use  high version agent plugins  with low version agent

### DISCUSSION THREAD

**Discussion by collery** [apache/skywalking#10775] [Q&A] [answered]
https://github.com/apache/skywalking/discussions/10686
hi,  in this Q. i found i need add extra plugins.  but my agent is only 8.9.0.
if i can import   8.15.0's plugins  into my plugins folder.

btw,if i can't .  Is it possible to directly upgrade the agent version？don't upgrade server version.

Looking forward to your reply  thks

**Comment by wu-sheng** [comment, marked as answer]
I would not recommend you mix agent core and plugin versions. That is not always compatible. 
About agent versions, for Java agents specifically, it is fine to use some previous releases. The basic features are working, but advanced features, such as profiling, may or may not work.

**Reply by collery** [reply]
thanks for your reply. i try see see
